test/test_CreateDecisionKnowledge.js

```diff
@@ -1,17 +1,108 @@
+      ' knowledge element.',
+    async () => {
+      const task = await createJiraIssue('Task', 'Dummy task for R2');
+      await axios.post(
+        `${JSONConfig.fullUrl}/rest/condec/latest/knowledge` +
+          `/createDecisionKnowledgeElement.json?idOfExistingElement=${task.id}` +
+          `&documentationLocationOfExistingElement=i`,
+        {
+          type: 'Issue',
+          projectKey: JSONConfig.projectKey,
+          description: 'Dummy decision knowledge issue for R2',
+          documentationLocation: 's',
+        },
+        localCredentialsObject
+      );
+      const taskAfterUpdate = await jira.findIssue(task.key);
+
+      chai
+        .expect(taskAfterUpdate.fields.comment.comments)
+        .to.be.an('Array')
+        .that.contains.something.like({
+          body: '{issue}\nDummy decision knowledge issue for R2{issue}',
+        });
+    }
+  );
+  it('(R3) A new alternative has the status "idea".', async () => {
+    const issue = await createJiraIssue('Issue', 'Dummy issue for R3');
+    await jira.addComment(issue.id, '{alternative}dummy alternative for R3{alternative}');
+    const knowledgeElements = await getKnowledgeElements();
+    chai.expect(knowledgeElements).to.be.an('Array').that.contains.something.like({
+      summary: 'dummy alternative for R3',
+      status: 'idea',
+      type: 'Alternative',
+    });
+  });
+  it('(R4) A new decision has the status "decided".', async () => {
+    const issue = await createJiraIssue('Issue', 'Dummy issue for R4');
+    await jira.addComment(issue.id, '{decision}dummy decision for R4{decision}');
+    const knowledgeElements = await getKnowledgeElements();
+    chai.expect(knowledgeElements).to.be.an('Array').that.contains.something.like({
+      summary: 'dummy decision for R4',
+      status: 'decided',
+      type: 'Decision',
+    });
   });
-  it('should create a decision knowledge issue via the create issue interface', async () => {
+  it(
+    '(R5) A new issue (=decision problem), i.e. an issue without linked decision has' +
+      ' the status "unresolved".',
+    async () => {
+      await createJiraIssue('Issue', 'Dummy issue for R5');
+      const knowledgeElements = await getKnowledgeElements();
+      chai.expect(knowledgeElements).to.be.an('Array').that.contains.something.like({
+        summary: 'Dummy issue for R5',
+        status: 'unresolved',
+        type: 'Issue',
+      });
+    }
+  );
+  xit( // This will be tested elsewhere
+    '(R6) A Jira issue (i.e. a decision knowledge element documented as an entire Jira issue) can' +
+      ' only be created in a view on the knowledge graph if the user has the rights to create Jira' +
+      ' issues (CONDEC-852, integrity).'
+  );
+
+  // This will be tested elsewhere
+  xit('(R7) If the webhook is activated, it will be fired (CONDEC-185).');
+
+  it('(E1) A decision knowledge element with the same id already exists.');
```

Comment:
@kleebaum Is it even possible to trigger this exception? And if yes, how? 😅

Reply:
@aaaailuj, good question. Please change the exception to "(E1) A decision knowledge element with the same summary and description already exists." and leave it out from testing (because for PartOfJiraIssueText objects this cannot be tested from UI and for Jira issues this is not implemented yet).